src/components/SearchBar.tsx

```diff
@@ -12,80 +12,73 @@ import SearchBarNativeComponent, {
 } from '../fabric/SearchBarNativeComponent';
 
 export const NativeSearchBar: React.ComponentType<SearchBarProps> &
-  typeof NativeSearchBarCommands = SearchBarNativeComponent as any;
+  typeof NativeSearchBarCommands =
+  SearchBarNativeComponent as unknown as React.ComponentType<SearchBarProps> &
+    SearchBarCommandsType;
 export const NativeSearchBarCommands: SearchBarCommandsType =
-  SearchBarNativeCommands as any;
+  SearchBarNativeCommands as SearchBarCommandsType;
+
+type ViewRef = React.ElementRef<typeof NativeSearchBar>;
 
 type SearchBarCommandsType = {
-  blur: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  focus: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  clearText: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
-  toggleCancelButton: (
-    viewRef: React.ElementRef<typeof NativeSearchBar>,
-    flag: boolean
-  ) => void;
-  setText: (
-    viewRef: React.ElementRef<typeof NativeSearchBar>,
-    text: string
-  ) => void;
-  cancelSearch: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
+  blur: (viewRef: ViewRef) => void;
+  focus: (viewRef: ViewRef) => void;
+  clearText: (viewRef: ViewRef) => void;
+  toggleCancelButton: (viewRef: ViewRef, flag: boolean) => void;
+  setText: (viewRef: ViewRef, text: string) => void;
+  cancelSearch: (viewRef: ViewRef) => void;
 };
 
-class SearchBar extends React.Component<SearchBarProps> {
-  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
-
-  constructor(props: SearchBarProps) {
-    super(props);
-    this.nativeSearchBarRef = React.createRef();
-  }
+function ImplSearchBar(props: SearchBarProps, ref: Ref<SearchBarCommands>) {
```

Comment:
Let's use `React.Ref` instead of pure `Ref`.

Reply:
https://github.com/software-mansion/react-native-screens/pull/2132/commits/6dcfb1841b98a3babbd19d679164b6cfea00fb9c